doc/source/changes/version_0_27.rst.inc

```diff
@@ -61,6 +61,42 @@ Miscellaneous improvements
 
 * added support for coloring numeric values for object arrays (e.g. arrays containing both strings and numbers).
 
+* allowed to sort values of an array along an axis (closes :issue:`225`):
+    >>> a = LArray([[10, 2, 4], [3, 7, 1]], "sex=M,F; nat=EU,FO,BE")
+    >>> a
+    sex\\nat  EU  FO  BE
```

Comment:
Extra `\` between axes names or missing one in the following examples. I don't remember if we need the second \ in the changelog but it should at least be consistent.